make/osde2e.mk

```diff
@@ -15,4 +15,4 @@ image/osde2e/build/push: image/osde2e/build image/osde2e/push
 
 .PHONY: test/compile/osde2e
 test/compile/osde2e:
-	CGO_ENABLED=0 go test -v -c -o managed-api-test-harness.test ./test/osde2e -ginkgo.noColor
```

Comment:
This change seems unrelated to Go version bump,
highlighting in case it was unintentionally added to the commit.

Reply:
On go 1.16, go test will not compile if extra flags are give to the `-c` flag. This was an intentionally commit.

Reply:
Okay, good to know.
Thank you for explanation 👍